### PR TITLE
chore(docs): add dockerfile to run docs

### PR DESCRIPTION
**Pull request by Hyperkid123**

### CODE REVIEW

File: packages/docs/docker.package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "next": "^10.0.7",
+    "prism-react-renderer": "^1.1.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-json-view": "^1.19.1",
+    "react-redux": "^7.2.2",
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
```
Comment (rvsia) [inline packages/docs/docker.package.json:29]: I don't think there is a difference between devDeps and deps. 🤔  Maybe we could merge them together.

File: packages/docs/docker.package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "concurrently": "^5.3.0",
+    "file-loader": "^6.2.0",
+    "jsdoc": "^3.6.6",
+    "jsdoc3-parser": "^2.0.0",
+    "next-transpile-modules": "^6.0.0",
+    "raw-loader": "^4.0.2",
+    "react-docgen": "^5.3.1",
+    "react-jss": "^10.5.0",
+    "url-loader": "^4.1.1",
+    "webpack": "^4.0.0"
```
Comment (rvsia) [inline packages/docs/docker.package.json:43]: Do you need to specify webpack? Isn't it a part of Next?

Comment (Hyperkid123) [reply packages/docs/docker.package.json:43]: Yes, we do. There is an issue with webpack 5 and one of the CSS decorators. I don't know what is it exactly so we need webpack 4 to be installed.

File: packages/docs/docker.package.json (hunk 1 above)
Comment (Hyperkid123) [reply packages/docs/docker.package.json:29]: Nah keeping it separate is good for our sanity.

**Review by rvsia — APPROVED** [review]

**Review by ryelo — APPROVED** [review]

**Pull request merged**